### PR TITLE
added required steps if you use a custom domain

### PR DESCRIPTION
**Pull request by zevaverbach**
Fixed a few typos too, and mentioned you have to log in to see the blockchain sync status. 

Without these additional steps, you get a 503 error when visiting the domain.

Note: I didn't test using a generated domain, so that may also be broken for all I know.

### CODE REVIEW

**Review by dennisreimann — APPROVED** [review]

**Comment by dennisreimann** [comment]
Thanks for the contribution, this looks good to me.

File: docs/LunaNodeWebDeployment.md [hunk 1]
```diff
@@ -29,6 +31,21 @@ Once your account has been verified, and credits added, go to the API section an
...
 5. Click Launch VM. Wait 6-7 minutes for Virtual Machine deployment.
-6. Visit the domain.
+
+If you've used a custom domain,
+
+6. SSH into the VM using either the password generated by LunaNode or your private-public key pair.
+7. Run the following commands.
+```bash
+$ sudo su -
+$ export BTCPAY_HOST=btc.helpers.fun
```
Comment (pavlenex) [inline docs/LunaNodeWebDeployment.md:41]: please use generic domain name here

**Review by pavlenex — CHANGES_REQUESTED** [review]

File: docs/LunaNodeWebDeployment.md (hunk 1 above)
Comment (zevaverbach) [reply docs/LunaNodeWebDeployment.md:41]: ooh, good catch

File: docs/LunaNodeWebDeployment.md [hunk 2]
```diff
@@ -2,9 +2,11 @@
 
 This article explains the **deployment of BTCPay Server through LunaNode Web-Wizard**. [LunaNode](http://lunanode.com/) is Canadian based hosting provider that accept Bitcoin payments and requires no personal identification except for a phone number verification.
 
-Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you do not have any technical knowledge. For the price of around US$ 8.8 per month, you can have a **self-hosted BTCpay**, which includes a Bitcoin full node and Lightning Network node.
+Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you don't have a lot of technical knowledge. However, if you set up a custom domain you will need to have some comfort with the shell.
```
Comment (Zaxounette) [inline docs/LunaNodeWebDeployment.md:5]:
```suggestion
Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you don't have a lot of technical knowledge. However, if you set up a custom domain you will need to have some comfort with the command line interface.
```

Comment (Zaxounette) [reply docs/LunaNodeWebDeployment.md:5]: Just making CLI generic instead of Shell specific. 
An unfamiliar user wouldn't know what shell is.

**Review by Zaxounette — COMMENTED** [review]
Just a comment to ask how crucial these command steps are ?

Because I've deployed multiple instances through the Lunanode Webwizard, with custom domain, and have never had to go CLI. The web-wizard includes the custom domain initial config.

**Comment by zevaverbach** [comment]
> Just a comment to ask how crucial these command steps are ?
> 
> Because I've deployed multiple instances through the Lunanode Webwizard, with custom domain, and have never had to go CLI. The web-wizard includes the custom domain initial config.

@Zaxounette it didn't work for me without those steps, which I gathered from [here](https://github.com/btcpayserver/btcpayserver-docker/issues/52).

**Review by Zaxounette — APPROVED** [review]

File: docs/LunaNodeWebDeployment.md [hunk 3]
```diff
@@ -2,9 +2,11 @@
 
 This article explains the **deployment of BTCPay Server through LunaNode Web-Wizard**. [LunaNode](http://lunanode.com/) is Canadian based hosting provider that accept Bitcoin payments and requires no personal identification except for a phone number verification.
 
-Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you do not have any technical knowledge. For the price of around US$ 8.8 per month, you can have a **self-hosted BTCpay**, which includes a Bitcoin full node and Lightning Network node.
+Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you don't have a lot of technical knowledge. However, if you set up a custom domain you will need to have some comfort with the command line interface.
```
Comment (britttttk) [inline docs/LunaNodeWebDeployment.md:5]:
```suggestion
Their web-wizard is one of the easiest ways to **deploy BTCPay Servers from a very user-friendly interface**. This method is highly recommended if you don't have a lot of technical knowledge. LunaNode will provide you with a generic domain for your server to get you started. If you want to set up a custom domain you will need to have some comfort with the command line interface. 
```

Comment (britttttk) [reply docs/LunaNodeWebDeployment.md:5]: Important to mention this, so that we don't intimidate non-technical users.

**Review by britttttk — CHANGES_REQUESTED** [review]

**Review by zevaverbach — COMMENTED** [review]
makes good sense to me!

**Pull request merged**